### ISSUE TITLE
Lack of visual feedback when login fails

### ISSUE THREAD

**Issue by olivierobert** [liamstevens111/django-google-search-data-viewer#8]
## Issue

When a user submits incorrect login information, the login page simply reloads.

## Expected

An error should be displayed so that the user understands that an error occurred.

## Steps to reproduce

1. Go to http://127.0.0.1:8000/auth/login/
2. Enter an incorrect email or password
3.Click on the button "Sign In"

**Comment by liamstevens111** [comment]
Yes, I was also aware that the e-mail input type on the login page is rendered as a **text** widget instead of **email** so there will also be no client-side validation to ensure the email is in its correct format. I think the field being rendered as text is because the default form that I have subclassed initially deals with username rather than emails.

 Will attend to both of these issues now :thumbsup:

**Comment by liamstevens111** [comment]
I have fixed this and it now correctly shows form errors upon entering invalid details. This was because when using **django-crispy-forms** to display the individual fields for extra customization it was not rendering the non-field errors which would normally be rendered from Django when including the entire from. Solution was to either to include the entire form or render (and style) the error fields individually so I decided to just render the form in its entirety. The pull request can be found [here](https://github.com/liamstevens111/google-search-data-viewer/pull/14)

I additionally modified the widget for the email field from **text** to **email**, although hesitant to do so due to potential reliance by Django's AuthenticationForm. After reading the source code from the form I have decided to go with the change due to no apparent issues caused by this. The pull request can be found [here](https://github.com/liamstevens111/google-search-data-viewer/pull/15)
